pkg/asset/agent/image/agentimage.go

```diff
@@ -263,15 +271,19 @@ func (a *AgentImage) PersistToFile(directory string) error {
 		if err != nil {
 			return err
 		}
-		logrus.Infof("Generated minimal ISO at %s", agentIsoFile)
+		msg = fmt.Sprintf("Generated minimal ISO at %s.", agentIsoFile)
 	} else {
 		// Generate full ISO
 		err = isoeditor.Create(agentIsoFile, a.tmpPath, a.volumeID)
 		if err != nil {
 			return err
 		}
-		logrus.Infof("Generated ISO at %s", agentIsoFile)
+		msg = fmt.Sprintf("Generated ISO at %s.", agentIsoFile)
+	}
+	if a.imageExpirsAt != "" {
+		msg = fmt.Sprintf("%s The ISO is valid up to %s", msg, a.imageExpirsAt)
```

Comment:
nit: it would be better if we added the period here, rather than above.
```suggestion
		msg = fmt.Sprintf("%s. The ISO is valid up to %s", msg, a.imageExpirsAt)
```